docs/src/components/programs.md

```diff
@@ -0,0 +1,11 @@
+# Programs
+
+- **Valence Program:** A program is an instance of the Valence Protocol. It is a particular arrangement and configuration of _ValenceAccounts_ and _Function_ across multiple _domains_. Programs are also associated with a set of _Subroutines_ that can be executed on the _Program_. For example, a POL lending relationship between two parties may be set up as a program.
+
+- **Domain:** Environments where _ValenceAccounts_ or _Libraries_ can be instantiated. Domains are defined by three properties:
+  1. Chain _(e.g., Neutron, Osmosis, Ethereum mainnet)_
+  2. Execution environment _(e.g., CosmWasm, EVM, SVM)_
+  3. Bridge from main domain _(e.g., Polytone over IBC, Hyperlane)_
```

Comment:
I think bridge should not be part of the domain definition.

Main reason is that bridge is a list of open channels of communications that a domain has, a domain can have 2 channel of communication (polytone and hyperlane), so other domains can talk with this domain over those 2 channels.
It doesn't mean we have 2 different domains, rather 1 domain with multiple channels.

Reply:
I'm still confused by this, on some conversations we had we included the Bridge in the domain in others we didn't. The truth is right now we are including the bridging configuration when we add an external domain to the authorization contract. So if for example we add Osmosis with Hyperlane and Osmosis with IBC they would be represented as different external domains. Prob need more clarification on this @uditvira